### PR TITLE
Chore(docs): Add guidance about deprecated deployment methods and intended support

### PR DESCRIPTION
**Pull request by eeaton**
Warnings about jenkins, CSR for new organizations.

I raised this issue in #1275 1275. Quick fix until we can properly address this in #1249

### CODE REVIEW

**Comment by eeaton** [comment]
Will add a doc fix for #1239 as well because it touches on similar issues

File: 0-bootstrap/README.md [hunk 1]
```diff
@@ -58,6 +58,20 @@ file.
 
 The purpose of this step is to bootstrap a Google Cloud organization, creating all the required resources and permissions to start using the Cloud Foundation Toolkit (CFT). This step also configures a [CI/CD Pipeline](/docs/GLOSSARY.md#foundation-cicd-pipeline) for foundations code in subsequent stages. The [CI/CD Pipeline](/docs/GLOSSARY.md#foundation-cicd-pipeline) can use either Cloud Build and Cloud Source Repos or Jenkins and your own Git repos (which might live on-premises).
 
+## Intended usage and support
+
+This repository is intended as an example to be forked, tweaked, and maintained in the user's own version-control system; the modules within this repository are not intended for use as remote references.
+Though this blueprint can help accelerate your foundation design and build, we assume that you have the engineering skills and teams to deploy and customize your own foundation based on your own requirements.
+
+We will support:
+ - Code is semantically valid, pinned to known good versions, and passes terraform validate and lint checks
```
Comment (apeabody) [inline 0-bootstrap/README.md:67]: nit: proper noun?

```suggestion
 - Code is semantically valid, pinned to known good versions, and passes Terraform validate and lint checks
```
```suggestion
 - Code is semantically valid, pinned to known good versions, and passes terraform validate and lint checks
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
+
+The intended audience of this blueprint is large enterprise organizations with a dedicated platform team responsible for deploying and maintaining their GCP environment, who is commited to separation of duties across multiple teams and managing their environment solely through version-controlled Infrastructure as Code. Smaller organizations looking for a turnkey solution might prefer other options such as [Google Cloud Setup](https://console.cloud.google.com/cloud-setup/overview)
+
+## Intended usage and support
+
+This repository is intended as an example to be forked, tweaked, and maintained in the user's own version-control system; the modules within this repository are not intended for use as remote references.
+Though this blueprint can help accelerate your foundation design and build, we assume that you have the engineering skills and teams to deploy and customize your own foundation based on your own requirements.
+
+We will support:
+ - Code is semantically valid, pinned to known good versions, and passes terraform validate and lint checks
```
Comment (apeabody) [inline README.md:14]:
```suggestion
 - Code is semantically valid, pinned to known good versions, and passes Terraform validate and lint checks
```

**Review by apeabody — APPROVED** [review]
Already reviewed by @daniel-cit

**Pull request merged**